### ISSUE TITLE
Find interaction vector for aromatic interactions. 

### ISSUE THREAD

**Issue by noahharrison64** [chemosim-lab/ProLIF#65]
Hi Cédric, 

First off this isn't an issue, rather I'm asking advice. The question is a little abstract I'm wondering whether you'll be able to make any suggestions for how to achieve the following, perhaps using some of the utils you have implemented here:

Hypothetical situation: Suppose I have 2 PDB files, both of the same protein but resolved with different ligands.
In PDB A there is a PiStacking interaction between Tyrosine54 sidechain and a benzene group of my ligand. 
In PDB B this interaction doesn't exist.

Would I be able to work out the vector between the centroids of the Tyr54 and benzene in PDB A, and then apply this vector to Tyr54 in PDB B, effectively allowing me to find the position of where the benzene could be positioned in this PDB B? 

I hope this makes sense, would be interested in hearing your thoughts!
Thanks,

Noah

**Comment by cbouy** [comment]
Hi Noah,

Did you close this issue because you found a solution to your problem, or should I convert it to a discussion instead of an issue?

**Comment by noahharrison64** [comment]
Hi Cédric, 

I actually closed this issue because I thought I could just try and work it out myself. I came to the solution that it perhaps was not possible to work out the centroid coordinates for the ligand. I'm not 100% certain but I believe that if you were to try and solve the equation relating the vectors, you would not be left with a single centroid coordinate. Instead, there are a circle of points that satisfy the equations. However if you think that perhaps a solution can be figured out then I'd love to hear your thoughts.

Instead I chose to align the PDBs and thus the benzene centroid coordinates should be somewhat transferable between PDBs, if that makes sense.

Best,
Noah
